pino.js

```diff
@@ -321,6 +325,10 @@ function onExit (fn) {
   }
 }
 
+// added private disable level
```

Comment:
if it's private, should we prefix a _, also make it non-enumerable?

also suggest `silent` - which makes more sense in the level context